### PR TITLE
Update google-nik-collection.rb to FREE 1.2.1

### PR DESCRIPTION
**Pull request by BenjaminHCCarr**
Google released the NIk Collection for free today.

See: https://plus.google.com/+NikCollection/posts/AFGsG2Di7EK

### CODE REVIEW

File: Casks/google-nik-collection.rb [hunk 1]
```diff
@@ -1,8 +1,8 @@
 cask 'google-nik-collection' do
-  version :latest
-  sha256 :no_check
+  version '1.2.1'
+  sha256 '7a06f90c8c6a9f09dc882dfb7a84cd4d97dae39e064540ecc2f5daa87569df2b'
 
-  url 'https://dl.google.com/dl/edgedl/photos/nikcollection-demo.dmg'
+  url 'https://dl.google.com/edgedl/photos/nikcollection-full-1.2.11.dmg'
```
Comment (vitorgalvao) [inline Casks/google-nik-collection.rb]: `url "https://dl.google.com/edgedl/photos/nikcollection-full-#{version}.dmg"`

File: Casks/google-nik-collection.rb [hunk 2]
```diff
@@ -1,8 +1,8 @@
 cask 'google-nik-collection' do
-  version :latest
-  sha256 :no_check
+  version '1.2.1'
```
Comment (vitorgalvao) [inline Casks/google-nik-collection.rb]: `version '1.2.11'`

File: Casks/google-nik-collection.rb (hunk 1 above)
Comment (BenjaminHCCarr) [inline Casks/google-nik-collection.rb]: Ah, I see you have done this before, thanks for cleaning it up, and the google/homebrew ruby editing. Very excited to see this committed.

Comment (vitorgalvao) [inline Casks/google-nik-collection.rb]: Actually, I was asking if you could make the changes.

**Comment by vitorgalvao** [comment]
Actually, this cask is also missing `uninstall`. There are [scripts](https://github.com/caskroom/homebrew-cask/tree/master/developer/bin) and [documentation](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/uninstall.md) geared for that.

Can you build one for it? Otherwise, it might just be best to remove it.

**Comment by BenjaminHCCarr** [comment]
Yes I will look into writing an uninstall script. I would love to have this
in homebrew.

I will make the additions, thanks for cleaning up/streamlining the formula.

**Comment by vitorgalvao** [comment]
Closing in favour of https://github.com/caskroom/homebrew-cask/pull/20081/, which has a few other enhancements. Thank you for your contribution.

**Pull request closed without merging**